storage/client_raft_test.go

```diff
@@ -863,5 +864,6 @@ func TestRaftAfterRemoveRange(t *testing.T) {
 		t.Fatal(err)
 	}
 	// Execute another replica change to ensure that MultiRaft has processed the heartbeat just sent.
+	// TODO(tschottdorf): without this line, test passes. Gotta investigate.
```

Comment:
it just never finds the new replica.

`V(1)`: https://gist.github.com/tschottdorf/3380546a93f8851e5bbd
`V(5)`: https://gist.github.com/tschottdorf/be8122421d3541e25aa1

for reference, here's the `V(1)` of current master.
https://gist.github.com/tschottdorf/dc7f5cae1de7d99bf41c